docs/source/examples.index.md

```diff
@@ -7,7 +7,6 @@ Examples often show examples of more than one thing. Below is a list of common o
 ## Evaluate structures with a calculator
  
 - ORCA: [ORCA via python script](examples.orca_python.md)
-- MACE: [Parallelize MACE calculator](examples.mace.md)
```

Comment:
Just noticed that you removed the MACE example.  Why?

Reply:
I have added it a long time ago and haven't kept it up to date; there's nothing special about parallelising MACE; MACE has its own documentation that will be more up-to-date. Though I maybe should have left it alone, on a better look there's not that much of a difference compared to the MACE documentation.

Reply:
OK, that's fine.